spec/cantaloupe_spec.rb

```diff
@@ -3,38 +3,38 @@
 require 'spec_helper'
 
 # Define some basic variables
-reg_username = ENV.has_key?('REG_USERNAME') ? ENV['REG_USERNAME'] + '/' : ''
+reg_username = ENV.key?('REG_USERNAME') ? ENV['REG_USERNAME'] + '/' : ''
 version = ENV['CANTALOUPE_VERSION']
 docker_env = { 'ENDPOINT_ADMIN_SECRET' => 'secret', 'ENDPOINT_ADMIN_ENABLED' => 'true' }
 dockerfile = create_test_dockerfile(reg_username)
-image_tag = reg_username + 'cantaloupe_' + version 
+image_tag = reg_username + 'cantaloupe_' + version
 
 # First we build either a 'stable' or 'dev' Cantaloupe image, depending on our ENV property
 if version == 'stable'
   expected_version = '4.0.3'
 
   describe docker_build('.', tag: image_tag)
 elsif version == 'dev'
-  commit_ref = ENV.has_key?('COMMIT_REF') ? '--build-arg COMMIT_REF=' + ENV['COMMIT_REF'] : ''
+  commit_ref = ENV.key?('COMMIT_REF') ? '--build-arg COMMIT_REF=' + ENV['COMMIT_REF'] : ''
   expected_version = '4.1-SNAPSHOT'
 
   # Build the _dev version of cantaloupe by calling docker here via a system call
   # this is required because we need to use --build-arg, and DockerSpec does not
   # currently support --build-ARG, see https://github.com/zuazo/dockerspec/issues/14
-  system("docker build --build-arg 'CANTALOUPE_VERSION=" + version + "' " + commit_ref + " -t " + image_tag + " .")
+  system('docker build --build-arg \'CANTALOUPE_VERSION=latest\' ' + commit_ref + ' -t ' + image_tag + ' .')
```

Comment:
This needs to be the `version` form. We're no longer using `latest` but have switched over to `dev` for consistency. The double quotes can be changed to single quotes though.